source/postcard-c/postcard.h

```diff
@@ -0,0 +1,1184 @@
+#ifndef POSTCARD_H
```

Comment:
Top of header documentation should probably include a link to the specification.